sass/pages/_homepage.scss

```diff
@@ -70,3 +70,122 @@
     }
   }
 }
+
+.price {
+  background-color: $color-background-light;
+  padding: 5rem 4rem;
+
+  &-heading {
+    width: 60%;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: 1fr max-content 1fr;
+    gap: 1.5rem;
+    align-items: center;
+    justify-content: center;
+
+    &::before,
+    &::after {
+      content: "";
+      display: block;
+      height: 1px;
+      background-color: $color-primary-light-1;
+    }
+  }
+
+  &-box {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(20rem, 30rem));
+    justify-content: center;
+    margin: 10rem 0;
+  }
+
+  &-table {
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
As a more general rule I think the first elements of a class should be the ones that define the container shape  and  then more specific elements, so for this class  I would say a order would be:
```css
&-table {
display: flex;
flex-direction: column;
height: 50rem;
border-radius: 10px;
align-items: center;
background-color: #fff;
overflow: hidden;
transition: all 0.2s;
 box-shadow: 0 1rem 1rem rgba(0, 0, 0, 0.158);
}
```

This is a suggestion but I think you can get the idea. I have created a followup issue to do this refactor so you do not need to do this refactor as part for this pull request. See #10

Reply:
Yeah, great suggestion